topo/node/nokia/nokia.go

```diff
@@ -328,9 +328,6 @@ func (n *Node) Delete(ctx context.Context) error {
 }
 
 func defaults(pb *tpb.Node) *tpb.Node {
-	if pb.Config == nil {
```

Comment:
some vendors add more or less labels, lets make it a standard to at least add labels for model, os, and vendor

Reply:
Fixed. Added OS and Model info for all vendors.